core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java

```diff
@@ -664,7 +663,6 @@ public LockResource lockWorkerMeta(EnumSet<WorkerMetaLockSection> lockTypes, boo
    *
    * @return the count
    */
-  @VisibleForTesting
```

Comment:
?

Reply:
hmm seems i forgot to mention this method needs locking too, will add to the comment